src/Modal.php

```diff
@@ -94,7 +94,7 @@ public function enableCallback()
             if ($this->cb->triggered() && $this->fx) {
                 $this->fx[0]($this->cb_view);
             }
-            $modalName = isset($_GET['__atk_m']) ? $_GET['__atk_m'] : null;
+            $modalName = $_GET['__atk_m'] ?? null;
```

Comment:
this change behavior if `$_GET['__atk_m'] == 0` (shouldn't be, but still)

Reply:
@DarkSide666 Really? `isset` and `??` should work the same.

Reply:
i think it should still work identical. ?? implies isset.

Reply:
unless I'm wrong and it was ?: ... :O